packages/flutter/lib/src/painting/basic_types.dart

```diff
@@ -269,3 +269,27 @@ bool axisDirectionIsReversed(AxisDirection axisDirection) {
   }
   return null;
 }
+
+/// Different ways to clip for a widget
+enum ClipOption {
```

Comment:
the word "option" here doesn't seem to add much. Is `Clip` ok?

Reply:
Yeah, that's Ok. Renaming now.